code/_globalvars/lists/maintenance_loot.dm

```diff
@@ -106,5 +106,10 @@ GLOBAL_LIST_INIT(maintenance_loot, list(
 	/obj/item/storage/toolbox/artistic = 2,
 	/obj/item/toy/eightball = 1,
 	/obj/item/reagent_containers/pill/floorpill = 1,
+	// yogs start
+	/obj/item/pod_parts/core = 1,
+	/obj/item/pod_parts/armor = 1,
```

Comment:
modularize

Reply:
that would require remaking the whole list.

Reply:
nah it requires adding to it mate